frontend/src/metabase/admin/datamodel/containers/MetadataEditorApp.jsx

```diff
@@ -48,7 +48,7 @@ const mapDispatchToProps = {
       : push(`/admin/datamodel/database/${id}`),
   selectTable: ({ id, db_id }) =>
     push(`/admin/datamodel/database/${db_id}/table/${id}`),
-  updateField: field => Fields.actions.update(field),
+  updateField: field => Fields.actions.updateField(field),
```

Comment:
I see we don't call too `Entity.actions.update` with lots of parameters directly, but we do that in the entity file.